.golangci.yml

```diff
@@ -25,6 +25,10 @@ linters-settings:
     settings:
       mnd:
         ignored-files: fake.go
+  forbidigo:
+    # Forbid the following identifiers (identifiers are written using regexp):
+    forbid:
+      - ^print.*$
```

Comment:
what should this catch? maybe this?
^fmt\.Print.*$ -- forbid use of Print statements because they are likely just for debugging

Reply:
this should catch all `printf("Test")` for example
The reason that I didn't used `^fmt.Print.*$` (it is the default) is that I use `fmt.Printf` in the version print command

Reply:
what is printf()? :)

Reply:
where do you see printf() ? :)